CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+Contribution Guide
+==================
+
+Discussing codestyle rules
+--------------------------
+
+If you want to discuss some of the codestyle rules, propose a new rule or suggest removing one, simply create a pull request or file an issue into the repository. Issues may be debating within one or two weeks, but in any case week after last comment in the discussion issue may be summarized by one of the maintainer of the repository.
```

Comment:
My formulation suggestions:
/may be debating/may be debated/
/week after last comment/after a week since the last comment/
/discussion issue/discussion/
/may be summarized/will be summarized/